src/System.Diagnostics.Debug/src/System/Diagnostics/DebuggerVisualizerAttribute.cs

```diff
@@ -11,36 +11,40 @@ namespace System.Diagnostics
     [AttributeUsage(AttributeTargets.Struct | AttributeTargets.Class | AttributeTargets.Assembly, AllowMultiple = true)]
     public sealed class DebuggerVisualizerAttribute : Attribute
     {
-        private readonly string _visualizerObjectSourceName;
-        private readonly string _visualizerName;
         private Type _target;
 
         public DebuggerVisualizerAttribute(string visualizerTypeName)
         {
-            _visualizerName = visualizerTypeName;
+            VisualizerTypeName = visualizerTypeName;
         }
+
         public DebuggerVisualizerAttribute(string visualizerTypeName, string visualizerObjectSourceTypeName)
         {
-            _visualizerName = visualizerTypeName;
-            _visualizerObjectSourceName = visualizerObjectSourceTypeName;
+            VisualizerTypeName = visualizerTypeName;
```

Comment:
Sorry I don't quite understand. What I did here was change the use of `private readonly` which is only used in the property to be a get only property. I don't think there's any changes in visibility - VisualizerTypeName remains public, its just that a couple of members were removed from the class.